### DISCUSSION TITLE
无法接收到部分频道的信息

### DISCUSSION THREAD

**Discussion by 1shin-7** [Mrs4s/go-cqhttp#1210] [Q&A] [answered]
在较大频道（似乎是5000人以上）中无法接收到任何文本信息，仅能接收到表情reaction更新消息；
![IMG_20211201_222544.jpg](https://user-images.githubusercontent.com/49867372/144461593-8a1791e9-8028-4cfa-99cb-102bc03079a8.jpg)
而在较少人数的频道中表现正常
附图为人数![IMG_20211203_002449.jpg](https://user-images.githubusercontent.com/49867372/144462088-b6d17dad-8d3a-4c93-86e5-b936feca7d91.jpg)

**Comment by Mrs4s** [comment, marked as answer]
最新的`dev`分支已经解决这个问题了, 可以等待下个版本更新

**Reply by 1shin-7** [reply]
尝试过使用actions中[CI#1503](https://github.com/Mrs4s/go-cqhttp/actions/runs/1514514591)， 但似乎仍然无效？|･ω･｀)

**Reply by Mrs4s** [reply]
https://github.com/Mrs4s/go-cqhttp/actions/runs/1526901905 昨天修的
